tools/jil/bin/server.js

```diff
@@ -15,15 +15,11 @@ config.cache = false
 var driver = require('../index')
 
 var hostname = config.host
-driver.router.handle(driver.assetServer.defaultAgentConfig.licenseKey, true)
+// driver.router.handle(driver.assetServer.defaultAgentConfig.licenseKey, true)
+// driver.assetServer.renderIndex = true
 
-driver.assetServer.renderIndex = true
-console.log('asset server: http://' + hostname + ':' + driver.assetServer.port)
-if (driver.assetServer.sslPort) {
-  console.log('asset server (SSL): https://' + hostname + ':' + driver.assetServer.sslPort)
-}
-console.log('fake router: http://' + hostname + ':' + driver.router.port)
-if (driver.router.sslPort) {
-  console.log('fake router (SSL): https://' + hostname + ':' + driver.router.sslPort)
-}
-console.log('secondary (cors) server: http://' + hostname + ':' + driver.assetServer.corsServer.port)
+setTimeout(() => {
```

Comment:
Minor: rather than waiting 500ms, can't you chain this to the `driver`'s `serverStartPromise` prop or `(on?)ready(cb)` func?

Reply:
Yea, just being lazy here :P